tmt/base.py

```diff
@@ -230,6 +230,21 @@ def _lint_summary(self):
             verdict(None, "summary should not exceed 50 characters")
         return True
 
+    def has_link(self, target):
+        """ Whether object contains specified link """
+        if isinstance(target, Link):
+            link = target.get()
+        else:
+            link_parts = target.split(':', maxsplit=1)
+            if len(link_parts) == 1:
+                link = Link(link_parts[0]).get()[0]
+            else:
+                link = Link({link_parts[0]: link_parts[1]}).get()[0]
+        for candidate in self._metadata.get('link', []):
+            if link == candidate:
```

Comment:
Would it make sense to support regexp search?

    tmt test show --link verifies:/spec/tests.*

Could be useful for searching for example feature areas.

Reply:
Good point. Now - should regexp work only on TARGET part?`--link .*:/spec/tests.*` might be handy. We could say that first `:` is used to split relation from target to make the split more deterministic (no troubles with escaping etc)

Reply:
Originally I thought mainly about the `target` part, but I like your idea. And it will allow to specify any relation as well. Nice!